### PR TITLE
Update CrSFML (binding) supported versions up to 2.6

### PR DESCRIPTION
**Pull request by oprypin**

### CODE REVIEW

File: download/bindings-fr.php [hunk 1]
```diff
@@ -55,7 +55,7 @@
         <tr>
             <th rowspan="1">Crystal</th>
             <td><a href="https://github.com/oprypin/crsfml" class="website"><img src="../images/bindings/crsfml.png" alt="CrSFML"/></a></td>
-            <td>2.5</td>
```
Comment (ChrisThrasher) [inline download/bindings-fr.php:58]: Why still bother mentioning anything but 2.6?

Comment (oprypin) [reply download/bindings-fr.php:58]: Not sure. The binding right below mentions `2.0 - 2.5`

Comment (oprypin) [reply download/bindings-fr.php:58]: Then maybe all occurrences should be removed

**Comment by eXpl0it3r** [comment]
Thanks for the update. The website has been updated accordingly.

**Pull request merged**